src/rockstor/system/users.py

```diff
@@ -100,6 +100,9 @@ def userdel(uname):
     except KeyError:
         # user doesn't exist
         return
+    #Ensure user get deleted from samba pass db
+    cmd = [SMBPASSWD, '-x', uname]
+    p = subprocess.Popen(cmd, shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Hi @MFlyer . Why not use run_command utility method instead of calling subprocess.Popen?